### ISSUE TITLE
[oj04fd] Element in sequential focus order has visible focus

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-board#93]
> [!NOTE]
> This issue is generated by a bot. Do not edit this issue body; leave comments instead.

**Status:** Blocked by issue

## Rule links

- [Proposed WAI rule](https://www.w3.org/WAI/standards-guidelines/act/rules/oj04fd/proposed/)
- [Approved WAI rule](https://www.w3.org/WAI/standards-guidelines/act/rules/oj04fd/)
- [Community Group rule file](https://github.com/act-rules/act-rules.github.io/blob/develop/_rules/sequentially-focusable-element-has-visible-focus-oj04fd.md)

## Complete implementations

- Alfa (semi-automated)
- Axe DevTools Pro
- SortSite
- Total Validator (+Browser)
- Trusted Tester
- UsableNet AQA - WCAG 2.2 AA Semi-auto

### Rule changes (6)

- [`b10d1ae`](https://github.com/act-rules/act-rules.github.io/commit/b10d1aeaf217a1eee18dff4101d1e373d8f88a70) Bibliography -> other resources (#2358\)
- [`2d8484b`](https://github.com/act-rules/act-rules.github.io/commit/2d8484b7565a0ac8beecf059856817594ea48a49) Revert "Change bibliography to other resources (#2350\)" (#2357\)
- [`4d39215`](https://github.com/act-rules/act-rules.github.io/commit/4d3921536af0d6899d150a42396d9ea3c06ff3ee) Change 'test cases' to 'examples' as required by ACT Rules Format 1.1 (#2352\)
- [`f8ace30`](https://github.com/act-rules/act-rules.github.io/commit/f8ace3038d2564711d54e39c2c443be678f66e1f) Change bibliography to other resources (#2350\)
- [`5ba46d9`](https://github.com/act-rules/act-rules.github.io/commit/5ba46d9e5777b96b0f94f4453a26052d1b460677) Update to rules format 1.1 (#2220\) — definitions: outcome
- [`346ae67`](https://github.com/act-rules/act-rules.github.io/commit/346ae67bf17cf9184743432d231e1b9779cd6e9b) WCAG 2.2 migration (#2147\)

### Definition-only changes (2)

- [`1bbef71`](https://github.com/act-rules/act-rules.github.io/commit/1bbef71c5e431f4db3ccd8b3d015766eab6ddd19) Minor typos correction and fixing broken links on the WAI website (#2336\) — definitions: outcome
- [`f9eace3`](https://github.com/act-rules/act-rules.github.io/commit/f9eace375b0d522108b1040de2c04c9610b979be) Tweak definition of focusable (#2210\) — definitions: focusable

## Open non-blocker Community Group issues

_None._

## Open blocker Community Group issues

- [#2368: Element in sequential focus order has visible focus \[oj04fd\] expectation allows common scenarios to pass even when failing](https://github.com/act-rules/act-rules.github.io/issues/2368)
